### PR TITLE
Refresh feature/grab script polish

### PR DESCRIPTION
**Pull request by sethalves**
-  #8214 with a refresh from master

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 52a7c8e032918ccf10e8da16067acf0c804759e5 available at: https://gist.github.com/5a6927cd3e8951ebf0978eb03bfd599c<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8224-52a7c8e.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8224-52a7c8e.dmg

File: scripts/system/controllers/handControllerGrab.js [hunk 1]
```diff
@@ -1388,19 +1373,24 @@ function MyController(hand) {
...
+                    // potentialFarTriggerEntity = entity;
                 }
             } else if (this.entityIsDistanceGrabbable(rayPickInfo.entityID, handPosition)) {
                 if (this.triggerSmoothedGrab() && !isEditing()) {
                     this.grabbedEntity = entity;
                     this.setState(STATE_DISTANCE_HOLDING, "distance hold '" + name + "'");
                     return;
                 } else {
-                    // TODO: highlight the far-grabbable object?
+                    // potentialFarGrabEntity = entity;
```
Comment (sethalves) [inline scripts/system/controllers/handControllerGrab.js]: What was intended with these 4 else clauses?

**Comment by sethalves** [comment]
Ryan QA'ed this

**Comment by RyanDowne** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;2m6D4Bp1&#x2F;60-equip-polish"> Equip polish</a>

**Pull request merged**